package.json

```diff
@@ -1,35 +1,63 @@
 {
-  "name": "progrrammers-only",
+  "name": "programmers-only",
   "version": "1.0.0",
   "description": "",
+  "keywords": [],
+  "license": "ISC",
+  "author": "",
   "main": "index.js",
   "scripts": {
+    "build": "tsc",
+    "lint": "eslint src/**/*.[jt]s",
+    "lint:fix": "eslint src/**/*.[jt]s --fix",
+    "start": "node dist/index.js",
     "test": "jest",
     "test:watch": "jest --watch"
   },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
+  "husky": {
```

Comment:
Now we have this configuration twice implemented (line 17 and line 58) move it to one `husky` property

Reply:
I check for that situations, but I miss it. Moved